src/sc4626.sol

```diff
@@ -22,48 +15,23 @@ abstract contract sc4626 is ERC4626, AccessControl {
 
         _grantRole(DEFAULT_ADMIN_ROLE, _admin);
         _grantRole(KEEPER_ROLE, _keeper);
-
-        treasury = _admin;
     }
 
-    bool flashLoanInitiated;
-    uint256 public performanceFee = 0.1e18;
-    uint256 public floatPercentage = 0.01e18;
-    address public treasury;
-
     /// Role allowed to harvest/reinvest
     bytes32 public constant KEEPER_ROLE = keccak256("KEEPER_ROLE");
 
-    event PerformanceFeeUpdated(address indexed user, uint256 newPerformanceFee);
-    event FloatPercentageUpdated(address indexed user, uint256 newFloatPercentage);
-    event TreasuryUpdated(address indexed user, address newTreasury);
+    bool public flashLoanInitiated;
 
-    modifier onlyAdmin() {
+    function _onlyAdmin() internal view {
         if (!hasRole(DEFAULT_ADMIN_ROLE, msg.sender)) revert CallerNotAdmin();
-        _;
     }
 
-    modifier onlyKeeper() {
+    function _onlyKeeper() internal view {
         if (!hasRole(KEEPER_ROLE, msg.sender)) revert CallerNotKeeper();
-        _;
-    }
-
-    function setPerformanceFee(uint256 newPerformanceFee) external onlyAdmin {
-        if (newPerformanceFee > 1e18) revert FeesTooHigh();
-        performanceFee = newPerformanceFee;
-        emit PerformanceFeeUpdated(msg.sender, newPerformanceFee);
-    }
-
-    function setFloatPercentage(uint256 newFloatPercentage) external onlyAdmin {
-        require(newFloatPercentage <= 1e18, "float percentage too high");
-        floatPercentage = newFloatPercentage;
-        emit FloatPercentageUpdated(msg.sender, newFloatPercentage);
```

Comment:
better not to change this file since it will break other strategies. if you need specific logic downstream in the inheritance chain override the functions or something instead

Reply:
oh i see, there is also scLiquity strat... fixed [here](https://github.com/lindy-labs/sandclock-contracts/pull/58/commits/d9c3117f365083a75722b3839f9ccebf7fcacff6)